model/old_architecture.py

```diff
@@ -0,0 +1,199 @@
+import torch
+from transformers import BertTokenizer, BertForSequenceClassification
+from transformers import BartForConditionalGeneration, BartTokenizer
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+from torch.utils.data import Dataset, DataLoader
+from transformers import AdamW
+from tqdm import tqdm
+import pandas as pd
+
+class MedicalDataset(Dataset):
```

Comment:
Class comments please! A quick couple lines for what this class is used for is enough.

Reply:
Done! Though I added it on the text_simplifier.py script instead of here